arisa.json

```diff
@@ -1,223 +1,231 @@
-{

```

Comment:
I think you have a different config for json formatting

Reply:
I think this might be because it was CRLF before and now changed to LF, like all other files.

Reply:
Yes, if you hide whitespace changes, you see what actually changed:
![image](https://user-images.githubusercontent.com/12124394/79105839-b9c9ad80-7d71-11ea-9baf-c7ef354ab216.png)

Reply:
Under Windows you can set `core.autocrlf` to `true`. This will make Git convert line endings to CRLF in the working directory and converts line endings of all added files to LF when added, see https://git-scm.com/docs/gitattributes#_end_of_line_conversion

Or we could add a `.gitattributes` file which specifies `* text=auto` so all files added are normalized (if they are detected as text) regardless of the `core.autocrlf` setting a contributor has.

Reply:
The file I committed is using LF now, before it was CRLF in the working dir for whatever reason.
All the other files did use LF already.